ftw/iframeblock/contents/iframeblock.py

```diff
@@ -41,11 +41,13 @@ class IIFrameBlockSchema(form.Schema):
                 default=u'Height calculation method'),
         description=_(
             u'iframeblock_height_calculation_method_desc',
-            default=u'See <a href="https://github.com/davidjbradshaw/iframe-resi'
-                    u'zer/tree/v3.5.5#heightcalculationmethod" target="_blank">'
-                    u'iframe-resizer package</a> for the available options.<br>'
+            default=u'See <a href="https://github.com/davidjbradshaw/iframe-'
+                    u'resizer/blob/v4.2.10/docs/parent_page/options.md#height'
+                    u'calculationmethod" target="_blank">iframe-resizer '
+                    u'package</a> for the available options.<br>'
```

Comment:
Maybe we can use this link? http://davidjbradshaw.github.io/iframe-resizer/#heightcalculationmethod
which seems better to me - but perhaps it not being https would scare some users??
```suggestion
            default=u'See <a href="http://davidjbradshaw.github.io/iframe-resizer/#heightcalculationmethod" ' 
                    u'target="_blank">iframe-resizer '
                    u'package</a> for the available options.<br>'
```

Reply:
I'd rather not use the io page because it is not version-specific. We have installed `4.2.9` so we should show the docs for this version. It might change in the future and then the user will be presented with options that are not possible.

Reply:
Ok - the thought occurred to me too.
The danger is (as happened last time) that an upgrade of resizer misses updating this link

Off by .1 error noted ;-)

Reply:
😋 Thanks!